docs/page.rst

```diff
@@ -930,47 +930,52 @@ In a nutshell, this is what you can do with PyMuPDF:
       pair: oc; insert_image
       pair: xref; insert_image
 
-   .. method:: insert_image(rect, filename=None, pixmap=None, stream=None, mask=None, rotate=0, alpha=-1, oc=0, xref=0, keep_proportion=True, overlay=True)
+   .. method:: insert_image(rect, *, alpha=-1, filename=None, height=0, keep_proportion=True, mask=None, oc=0, overlay=True, pixmap=None, rotate=0, stream=None, width=0, xref=0)
```

Comment:
What is the `*` parameter for?

Reply:
It's Python notation that means that the following args are keyword-only. For example trying to pass alpha=1 with `insert_image(myrect, 1)` will raise an error, one has to do `insert_image(myrect, alpha=1)`.